cmd/cql-minerd/node.go

```diff
@@ -73,3 +83,54 @@ func createServer(privateKeyPath, pubKeyStorePath string, masterKey []byte, list
 
 	return
 }
+
+func registerNodeToBP(timeout time.Duration) (err error) {
+	// get local node id
+	localNodeID, err := kms.GetLocalNodeID()
+	if err != nil {
+		err = errors.Wrap(err, "register node to BP")
+		return
+	}
+
+	// get local node info
+	localNodeInfo, err := kms.GetNodeInfo(localNodeID)
+	if err != nil {
+		err = errors.Wrap(err, "register node to BP")
+		return
+	}
+
+	log.WithField("node", localNodeInfo).Debug("construct local node info")
+
+	pingWaitCh := make(chan proto.NodeID)
+	bpNodeIDs := route.GetBPs()
+	for _, bpNodeID := range bpNodeIDs {
+		go func(ch chan proto.NodeID, id proto.NodeID) {
+			for {
+				select {
+				case <-ch:
+					return
+				case <-time.After(time.Second):
+				}
+
+				err := rpc.PingBP(localNodeInfo, id)
+				if err == nil {
+					ch <- id
+					return
+				} else {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
if block ends with a return statement, so drop this else and outdent its block